libevm/VM.cpp

```diff
@@ -815,7 +815,93 @@ void VM::interpretCases()
 			m_sp -= 2;
 		CASE_END
 
+		CASE_BEGIN(JUMPTO)
+		THROW_IF_SUBS_DISABLED();
+			ON_OP();
+			updateIOGas();
+			{
+				uint64_t dest       = m_code[m_pc++];
```

Comment:
I guess the new specification still assumes that bytecode is suffixed with an infinite number of zeros. Does the access here still work with our "enlarge the code just by a bunch of bytes" fix?